pueue/src/bin/pueue.nu

```diff
@@ -0,0 +1,191 @@
+module pueue_completions {
```

Comment:
Please move this file to a `utils/completions/` directory.

The `src` folder is reserved for rust code and things that're tightly integrated with that code.
the `src/bin` folder specifically should only contain entry points for binaries.